library/MedEx/MedEx_cron.php

```diff
@@ -0,0 +1,29 @@
+<?php
+    /**
+     * /library/MedEx/MedEx_cron.php
+     *
+     * This file is not active.
+     * To utilize this as a cron job, you should move it to a directory outside of the apache2 filepath.
+     * You will need to correct the directory links to the "require_once" files, depending on where you place them.
+     * Ensure the file is executable.
+     *
+     *  LARGE PRACTICES SHOULD DISABLE MedEx in background_services table and instead use this file w/ cron
+     *  every 5 minutes as a suggested frequency to ensure medex_outgoing table is up-to-date.
+     *
+     * @package MedEx
+     * @link    http://www.MedExBank.com
+     * @author  MedEx <support@MedExBank.com>
+     * @copyright Copyright (c) 2017 MedEx <support@MedExBank.com>
+     * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+     */
+    
```

Comment:
place following here (if still plan to keep this script):
```
// comment out below exit line to make script active
exit;
```

Reply:
Also place the following to ensure this is only run as a command line script:
```
if (php_sapi_name() !== 'cli') {
    exit;
}
```

Reply:
The background_services file specifically looks for a function in the file to execute.  Can cron do this or do I need to add something like this:

`if (php_sapi_name() == 'cli') {
    start();
}`

Reply:
And is this safe to do with a file sitting in an apache accessible directory?

Reply:
hi @ophthal ,
For any script that we do `$ignoreAuth=true;`, we need to be cautious (it is not a no-no, but we just need to be sure code is contained and secure). The php_sapi_name() check makes use feel better since it then can't be run from browser, so in theory should be ok to leave in apache public directory. That being said, if it is a script that can be moved, then that's even more secure :)
btw, are we talking about this script or the background_services script?

Reply:
Removing the cron script.
Will add directions in _background to set cron and for background_services::execute_interval to 5 minutes in the next commit.